apps/passport-client/src/state.ts

```diff
@@ -41,7 +40,12 @@ export interface AppState {
         revealed: boolean;
         refreshAll: () => Promise<void>;
       }
-    | { modalType: "frogcrypto-export-pcds" };
+    | { modalType: "frogcrypto-export-pcds" }
+    | {
+        modalType: "account-import";
+        imported?: number;
```

Comment:
Good to know about this capability, thanks.

Reply:
I think this change falls under the broad question of #1291 - effectively what you're doing is namespacing the `'imported'` and `'error'` fields, rather than putting them directly into `appState`. It's a reasonable thing to do, though personally I would have put this directly into `appState`. It may be necessary to do so if you choose to follow my suggestion regarding implementing this as a screen rather than a modal.